Template/Tag/MatomoTag.php

```diff
@@ -105,6 +106,34 @@ public function getParameters()
                     return trim($value);
                 };
             }),
+            $this->makeSetting('searchKeyword', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) use ($trackingType) {
+                $field->title = Piwik::translate('TagManager_SearchKeyWord');
+                $field->customFieldComponent = self::FIELD_VARIABLE_COMPONENT;
+                $field->description = Piwik::translate('TagManager_SearchKeyWordHelp');
+                $field->condition = 'trackingType == "search"';
+                if ($trackingType->getValue() === 'search') {
+                    $field->validators[] = new NotEmpty();
+                    $field->validators[] = new CharacterLength(0, 500);
+                }
+            }),
+            $this->makeSetting('searchCategory', 'false', FieldConfig::TYPE_STRING, function (FieldConfig $field) use ($trackingType) {
```

Comment:
Is there a specific reason you're using `'false'` as the default value, because it appears to be causing some tests to fail? Looks like you did it with `searchCount` as well.

Reply:
Ah. I see that the help text says to put `false` if you don't know. So, I guess you just need to update the XML that the tests compare against.